test/kafo/hook_context_test.rb

```diff
@@ -45,6 +46,25 @@ module Kafo
       end
     end
 
+    describe "#module_present?" do
+      specify do
+        kafo.expect :module, nil, ['unknown_module']
+        assert_equal context.module_present?('unknown_module'), true
+      end
+
+      specify do
+        assert_equal context.module_present?('nonexistent_module'), false
+      end
+
+      specify do
+        mod = Minitest::Mock.new
+        mod.expect :nil?, false
+        mod.expect :enabled?, true
```

Comment:
This isn't expected for `module_present?`. Overall I'm not sure what this test provides over the test on line 50.